spec/features/workflow_service_creation_spec.rb

```diff
@@ -1,25 +1,52 @@
 require 'spec_helper'
 
 RSpec.feature 'Workflow Service Creation' do
-  let(:current_user) do
-    double(
-      :webauth_user,
+  # let(:current_user) do
+  #   double(
+  #     :webauth_user,
+  #     login: 'sunetid',
+  #     logged_in?: true,
+  #     permitted_apos: [],
+  #     is_admin: true,
+  #     roles: []
+  #   )
+  # end
+  # before do
+  #   allow(current_user).to receive(:can_manage_object?).and_return(true)
+  #   allow_any_instance_of(ApplicationController)
+  #     .to receive(:current_user)
+  #     .and_return(current_user)
+  # end
+
+  before :each do
+    @druid = 'qq613vj0238'  # a fixture Dor::Item record
+    @pid = DruidTools::Druid.new(@druid).druid
+    @item = instantiate_fixture(@druid, Dor::Item)
+    allow(Dor).to receive(:find).with(@pid).and_return(@item)
+    webauth = double(
+      'WebAuth',
       login: 'sunetid',
       logged_in?: true,
-      permitted_apos: [],
-      is_admin: true,
-      roles: []
+      attributes: {'DISPLAYNAME' => 'Admin'}
     )
+    @current_user = User.find_or_create_by_webauth(webauth)
+    allow(@current_user).to receive(:groups).and_return(User::ADMIN_GROUPS)
+    allow_any_instance_of(ApplicationController)
+      .to receive(:current_user)
+      .and_return(@current_user)
   end
-  before do
-    allow_any_instance_of(ItemsController).to receive(:current_user)
-      .and_return(current_user)
-    allow_any_instance_of(CatalogController).to receive(:current_user)
-      .and_return(current_user)
-  end
-  scenario 'redirect and display on show page' do
-    visit add_workflow_item_path 'druid:qq613vj0238'
-    click_button 'Add'
+
+  scenario 'redirect and display on show page', js: true do
+    path = add_workflow_item_path(@pid)
+    visit path
+    expect(page).to have_content('Add workflow')
+    expect(page).to have_button('Add')
+    find('#wf').find(:option, 'accessionWF').select_option
+    find("#add_wf_button").trigger("click")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.